R/Achilles.R

```diff
@@ -631,6 +642,18 @@ achilles <- function (connectionDetails,
                                 achillesTables = unique(achillesTables))
   }
   achillesSql <- c(achillesSql, indicesSql)
+
+  if (optimizeAtlasCache) {
+    optimizeAtlasCacheSql <- optimizeAtlasCache(connectionDetails = connectionDetails,
+                                                resultsDatabaseSchema = resultsDatabaseSchema,
+                                                vocabDatabaseSchema = vocabDatabaseSchema,
+                                                outputFolder = outputFolder,
+                                                sqlOnly = sqlOnly,
+                                                verboseMode = verboseMode,
+                                                tempAchillesPrefix = tempAchillesPrefix)
+
+    achillesSql <- c(achillesSql, optimizeAtlasCacheSql)
+}
```

Comment:
Can you add a tab here to fix the indent?

Reply:
fixed